File: python/7_kyu/every_possible_sum_of_two_digits.py
# Every possible sum of two digits
# https://www.codewars.com/kata/5b4e474305f04bea11000148/train/python

def digits(num):
    num = list(str(num))
    arr = []
    
    for i in range(0, len(num)):
        for j in range(i + 1, len(num)):
            arr.append(int(num[i]) + int(num[j]))
    
    return arr
        

print(digits(12345))

File: javascript/7_kyu/every_possible_sum_of_two_digits_.js
// Every possible sum of two digits
// https://www.codewars.com/kata/5b4e474305f04bea11000148/train/javascript

function digits(num){
    var arr = Array.from(String(num), Number);
    var ans = [];

    for (var i = 0; i < arr.length; i++) {
        for (var j = i + 1; j < arr.length; j++) {
            ans.push(arr[i] + arr[j]);
        }
    }
    return ans;
}

console.log(digits(12345));